app/views/index.erb

```diff
@@ -19,30 +18,37 @@
       <form class="add-new" action="/comedians/new" method="get">
         <input type="submit" value="Submit">
       </form>
+    </section>
 
     <section id="statistics">
       <h1>Statistics</h1>
-      <h3>Average Comedian Age: <%= @comedians.average_age.round %></h3>
-      <h3>Total Specials: <%= @specials.count %></h3>
-      <h3>Average Run Time: <%= @specials.average_run_time.round %></h3>
-      <h4>Hometowns: <%= @comedians.hometowns %></h4>
+      <div class="three-stats">
+        <h3>Average Comedian Age: <%= @comedians.average_age.round %></h3>
+        <h3>Total Specials: <%= @specials.count %></h3>
+        <h3>Average Run Time: <%= @specials.average_run_time.round %></h3>
+      </div>
+      <div class="hometowns-text">
+        <h4>Hometowns: <%= @comedians.hometowns %></h4>
+      </div>
     </section><br/>
 
     <section id="profiles">
       <% @comedians.each do |comedian| %>
-      <div id=<%= "comedian-#{comedian.id}" %>>
-        <img src="<%= comedian.photo%>"
-        width = 200>
-      <h3><%= comedian.name %></h3>
-      <li><%= comedian.age %></li>
-      <li><%= comedian.home_town %></li>
-      <li><%= comedian.specials.count %></li><br/>
-        <% comedian.specials.each do |special| %>
-          <img src="<%= special.thumbnail %>"
-          width = 100>
-          <li>Name: <%= special.name %></li>
-          <li>Run Time: <%= special.run_time %></li><br/>
+        <div id=<%= "comedian-#{comedian.id}" %>>
+
+          <div class="card">
+            <img src="<%= comedian.photo%>" width = 200>
```

Comment:
html attributes should always have their values wrapped in quotes, so `width="200"` is preferred. You could also throw a CSS class in there like `class="thumbnail"` and set width/height/etc in CSS instead.